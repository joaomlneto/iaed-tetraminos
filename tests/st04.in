11
1 1 V
1 5 V
3 4 B
4 1 G
3 1 O
1 5 R
4 3 G
2 5 B
2 4 B
2 3 B
2 9 V
