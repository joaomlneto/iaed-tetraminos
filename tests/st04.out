9
3

|  BB      | 9
|  BB      | 8
|   BB     | 7
|   BB     | 6
|    BB    | 5
|    BB    | 4
|OOGGG     | 3
| OOGRRRR  | 2
|GGGBB     | 1
 ----------
